CodeStarterCamp_Week3/person.swift

```diff
@@ -16,7 +16,25 @@ class Person {
         self.name = name
     }
     
-    func buy(coffee: Coffee, at coffeeShop: CoffeeShop) {
-        coffeeShop.takeOrderFor(coffee: coffee)
+    func order(_ coffee: Coffee, at coffeeShop: CoffeeShop) {
+        if let coffeeCost = coffeeShop.menuPan[coffee] {
```

Comment:
guard let 옵셔널 바인딩을 활용해보시면 코드가 조금 더 간결해질 것 같네요 😊
```suggestion
        guard let coffeeCost = coffeeShop.menuPan[coffee] else {
            print("\(coffee.name)는 없는 메뉴 입니다.")
            return
        }
```